lambdas/data-migration2/README.md

```diff
@@ -1,14 +1,17 @@
 # Data Migration 2 Lambda
 
-## The second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.
+The data-migration2 lambda is the second of two Lambdas to be run in order to migrate existing DynamoDB data to PostgreSQL.
 
 ### Running the API
-This second Lambda in the data migration process can be run in two ways:
+This second Lambda in the data migration process can be run in several ways:
 
 1. In an ECS container using the Cumulus ECS Service
 2. Directly from the AWS console / command line
+3. By invoking an async operation using the provided `${PREFIX}-postgres-migration-async-operation` lambda and invoking from the AWS console or command line.
 
-Running the Lambda within an ECS container is the more common approach as it is likely this execution will take some time. See `/example/data-migration-ecs` for an example container configuration.
+Running the Lambda within an ECS container is a more common approach as it is likely this execution will take some time. See `/example/data-migration-ecs` for an example container configuration.
```

Comment:
This is inaccurate now. That example doesn't exist. People just run this Lambda and it transparently handles starting an async operation for them